.vsconfig

```diff
@@ -0,0 +1,6 @@
+{
+  "version": "1.0",
+  "components": [
+    "Microsoft.VisualStudio.Workload.ManagedGame"
+  ]
+}
```

Comment:
Some of these files should not be included into main. We'll see what's the problem tomorrow